src/player.cpp

```diff
@@ -734,11 +734,7 @@ bool Player::canOpenCorpse(uint32_t ownerId) const
 
 uint16_t Player::getLookCorpse() const
 {
-	if (sex == PLAYERSEX_FEMALE) {
-		return ITEM_FEMALE_CORPSE;
-	} else {
-		return ITEM_MALE_CORPSE;
-	}
+	return sex == PLAYERSEX_FEMALE ? ITEM_FEMALE_CORPSE : ITEM_MALE_CORPSE;
```

Comment:
Personally, I don't like the ternary operator. A switch case block would be prettier code here and already be more efficient since it (most certainly) will become a jump table (exact identical assembly to ternary operator)